introduction.md

```diff
@@ -1 +1,5 @@
 Hello my name is Michele. Codeschool git courses are great. and RUby too.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?